sdk/servicebus/service-bus/src/serviceBusMessage.ts

```diff
@@ -562,6 +614,29 @@ export function isServiceBusMessage(possible: unknown): possible is ServiceBusMe
   return isObjectWithProperties(possible, ["body"]);
 }
 
+/**
+ * @internal
+ * @ignore
+ */
+export function isAmqpAnnotatedMessage(possible: any): possible is AmqpAnnotatedMessage {
+  return (
+    possible != null &&
+    typeof possible === "object" &&
+    "body" in possible &&
+    "bodyType" in possible &&
+    possible.constructor.name !== ServiceBusMessageImpl.name
+  );
+}
```

Comment:
Keep the linter happy by updating to:
```suggestion
export function isAmqpAnnotatedMessage(possible: unknown): possible is AmqpAnnotatedMessage {
  return (isObjectWithProperties(possible, ["body", "bodyType"]) && possible.constructor.name !== ServiceBusMessageImpl.name);
}
```